app/models/review.rb

```diff
@@ -0,0 +1,20 @@
+# == Schema Information
+#
+# Table name: reviews
+#
+#  id            :integer          not null, primary key
+#  restaurant_id :integer
+#  reviewer_name :string
+#  rating        :integer
+#  comment       :text
+#  created_at    :datetime         not null
+#  updated_at    :datetime         not null
+#
+
+class Review < ApplicationRecord
+  belongs_to :restaurant
+
+  validates :reviewer_name, :presence => true
```

Comment:
use the easier to write(and read) way - presence: true 
(it replaces the => way of writing)

Reply:
Done. Thanks!